wrappers/core18_test.go

```diff
@@ -179,6 +182,9 @@ WantedBy=snapd.service
 	}, {
 		filepath.Join(dirs.SnapDBusSessionServicesDir, "io.snapcraft.SessionAgent.service"),
 		"[D-BUS Service]\nName=io.snapcraft.SessionAgent",
+	}, {
+		filepath.Join(dirs.SnapDesktopFilesDir, "io.snapcraft.SessionAgent.desktop"),
+		"[Desktop Entry]\nName=Snapd User Session Agent",
 	}} {
```

Comment:
should we have a check also for snap-handle-link.desktop now that the list is explicit?

Reply:
can we make this change?

Reply:
Added.